### DISCUSSION TITLE
post/11/

### DISCUSSION THREAD

**Discussion by giscus** [HoshinoRei/blog#5] [General]
# post/11/

今天小玲来介绍 Windows 上的又一个包管理器——Scoop，这也是小玲在 Windows 系统上最喜欢也是用的最多的包管理器。

https://blog.bling.moe/post/11/

**Comment by Wanderingkang** [comment]
和您商量个事呗，能不能开个nastools分支，加入个功能，新增在指定的时间与媒体服务器同步，在同步目录前给媒体服务器发送arp数据包或者唤醒数据包，延迟（时间）后开始同步。这样媒体服务器可以休眠。致使nastools 媒体服务器可以分离。谢谢哦

**Comment by wcmiracle** [comment]
陷入死胡同了,按照第一步更改完目录, scoop install安装软件的时候提示我找不到buckets 但是添加buckets又要安装git
